admin_guide/managing_networking.adoc

```diff
@@ -1204,8 +1204,12 @@ For example, to assign the `MyProject` project to an IP address of
 $ oc patch netnamespace MyProject -p '{"egressIPs": ["192.168.1.100"]}'
 ----
 +
-The `egressIPs` field is an array, but must be set to a single IP address. If
-setting multiple IPs, the other IPs will be ignored.
+The `egressIPs` field is an array. While in earlier releases it could only
+contain a single IP address, as of {product-title} version 3.10 `egressIPs` can
+be set to two or more IP addresses on different nodes to provide high
+availability. If multiple egress IP addresses are set, pods use the first IP in
+the list for egress, but if the node hosting that IP address fails, pods will
+switch to using the next IP in the list after a short delay.
```

Comment:
maybe `s/pods will switch to using the next/pods will use the next or /pods will switch to the next`